### PR TITLE
Upload classification and subject dumps as gzip

### LINKED ISSUE #1077: Compressing the csv data dumps for subjects and classifications
Future request - I'm noticing it's already starting to take minutes for the classification database for P4T to download. Also one of my collaborators mentioned that he uses safari and it automatically tries to load the csv when he clicks the link from his email since it's uncompressed  - eventually it might be better if these were compressed

### PR DESCRIPTION
**Pull request by edpaget**
Closes #1077

### CODE REVIEW

**Comment by edpaget** [comment]
This won't solve the automatically opening the csv in Safari issue. Maybe I should include a note to 'right-click select Save As' on the link in the email?

File: app/workers/concerns/dump_worker.rb [hunk 1]
```diff
@@ -14,9 +14,21 @@ def mailer
...
+
+  def csv_file_path
     "#{Rails.root}/tmp/#{project_file_path.join("_")}.csv"
   end
 
+  def gzip_file_path
+    "#{Rails.root}/tmp/#{project_file_path.join("_")}.gz"
+  end
+
+  def s3_object_path
```
Comment (camallen) [inline app/workers/concerns/dump_worker.rb]: is this used?

**Comment by camallen** [comment]
Looks good. just wondering about the empty method.

File: app/workers/concerns/dump_worker.rb (hunk 1 above)
Comment (edpaget) [inline app/workers/concerns/dump_worker.rb]: No

**Pull request merged**